packages/flutter/lib/src/widgets/scroll_metrics.dart

```diff
@@ -36,7 +36,7 @@ abstract class ScrollMetrics {
   ///
   /// This is useful if this object is mutable, but you want to get a snapshot
   /// of the current state.
-  ScrollMetrics cloneMetrics() => new FixedScrollMetrics.clone(this);
+  ScrollMetrics toMetrics() => new FixedScrollMetrics.clone(this);
```

Comment:
Oops, sorry, too tired 😴 . Basically methods with no parameter that only returns a new instance of any type built with a constructor that takes `this` as parameter.